pallets/messages/src/benchmarking.rs

```diff
@@ -42,7 +43,13 @@ fn ipfs_message<T: Config>(schema_id: SchemaId) -> DispatchResult {
 	let bounded_payload: BoundedVec<u8, T::MaxMessagePayloadSizeBytes> =
 		payload.try_into().expect("Invalid payload");
 
-	MessagesPallet::<T>::add_message(provider_id.into(), None, bounded_payload, schema_id)?;
+	MessagesPallet::<T>::add_message(
+		provider_id.into(),
+		None,
+		bounded_payload,
+		schema_id,
+		<T as frame_system::Config>::BlockNumber::one(),
+	)?;
```

Comment:
Maybe out of scope, but if not, let's consider adding an assertion to the benchmarks: 
- add_onchain_message
- add_ipfs_message

Reply:
It seems good practice to keep this assertion in the assertion section. I notice this update throughout substrates benchmarks.